src/styles/main.scss

```
@@ -0,0 +1,50 @@
+@import "./variables.scss";
+
+.clock {
+  border-radius: 50%;
+  position: relative;
+  width: $clock-size;
+  height: $clock-size;
+  background-color: rgb(10, 233, 151);
+  z-index: -2;
+
+  &__centr-dot {
+    @include centr-dot-size;
+    position: absolute;
+    background-color: rgb(218, 149, 2);
+    z-index: 0;
```

and just set it to 1 here
```suggestion
    z-index: 1;
```